templates/store/categories/iot.html

```diff
@@ -0,0 +1,456 @@
+{% extends webapp_config['LAYOUT'] %}
+
+{% block meta_copydoc %}{% endblock %}
```

Comment:
Please link the copy doc here: https://docs.google.com/document/d/1yfnQ5pXu5tDTG3M7Yv_gpjd894NUzSBKcvhB_pZqrMo/edit